src/Libraries/.editorconfig

```diff
@@ -2012,7 +2012,7 @@ dotnet_style_null_propagation = true
 # Title    : Use auto property
 # Category : Style
 # Help Link: https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0032
-dotnet_diagnostic.IDE0032.severity = warning
+dotnet_diagnostic.IDE0032.severity = silent
```

Comment:
I'm unsure why these 2 warnings are now popping up with the new SDK. Need to figure out why.

Reply:
Probably not updating the SDK, but likely updating arcade would do this, yes.

Reply:
https://github.com/dotnet/extensions/blob/6786d6d900ca18fb869baca49ea9d34972ff9f22/src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/CSharp/JsonSerialization/JsonUtilities.cs#L17-L18

is an example of a warning.